spaghetti/ffs.rb

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env ruby
+foo = ARGF.read
+puts foo.gsub(/\/\*([^*])/, '/**~SING \1').gsub(/\/\/(.*)/, '/**~LINE \1*/')
```

Comment:
I'd put these into a `migration` directory or something, instead of littering the root directory.